app.js

```diff
@@ -14,12 +14,32 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(__dirname + "/views/partials")
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => {
+    console.log('Beers from the database: ', beersFromApi)
+  res.render('beers', {beersList: beersFromApi});
+  })
+  .catch(error => console.log(error));
+});
+
+app.get('/random', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then(responseFromAPI => {
+    console.log(`Random beer: `, responseFromAPI);
+    res.render('random-beers', {randBeer: responseFromAPI});  
+  })
+  .catch(error => console.log(error));
+});
+
```

Comment:
Great job Mark! Keep it up! When you have some time you can go over the bonuses. I'm sure they will be no brainer for you ;)